### ISSUE TITLE
[Instance][WoTLK][Violent Hold] Zuramat the Obliterator

### ISSUE THREAD

**Issue by TheRealestMcCoy** [dalaranwow/dalaran-wow#4644]
**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Zuramat the Obliterator sentrys can stay after fight.
**Expected behaviour**: Tell us what should happen instead
After Zuramat the Obliterator dies the sentrys should all despawn.
**Steps to reproduce the problem**:

1. This is random if it happens 
2. It has happened twice.
3. Kill to fast can lead to this.

**Include proofs for this behaviour**
![wowscrnshot_032917_151431](https://cloud.githubusercontent.com/assets/12252441/24472380/82ef106a-1493-11e7-83e3-238a64059ede.jpg)
![wowscrnshot_032917_151429](https://cloud.githubusercontent.com/assets/12252441/24473051/feb5ff86-1495-11e7-96eb-63782ac35f5d.jpg)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.
[Void Sentry](http://www.wowhead.com/spell=54369/summon-void-sentry)
[Boss](http://www.wowhead.com/npc=29314/zuramat-the-obliterator)

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
Scripts/VioletHold: Zuramat - Despawn all Void Sentry creatures on death of the boss 

* closes: https://github.com/dalaranwow/dalaran-wow/issues/4644
* commit: e8b6062c13b3d52de07331114dd53a58b81c2f6c

Added a save despawn in the justdied function of the boss. They should all despawn now

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.
